update_compilers/install_assembly_compilers.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+ARG1="$1"
```

Comment:
is this still needed? Seems it's in `common.inc` ?

Reply:
The idea is that you can also run each file separatedly, and I did not find a better way to ensure I also can call `install_*_compilers.sh nightly` properly.
Would `. ${SOURCE_DIR}/common.inc "$@"` work?